cdap-ui/server/express.js

```diff
@@ -61,11 +62,31 @@ const getExpressStaticConfig = () => {
     maxAge: '1y'
   };
 };
+
+function getFaviconPath(uiThemeConfig) {
+  let faviconPath = DIST_PATH + '/assets/img/favicon.png';
+  let themeFaviconPath;
+  let themeFaviconPathType = objectQuery(uiThemeConfig, ['content', 'favicon', 'type']);
+  let themeFaviconPathUrl = objectQuery(uiThemeConfig, ['content', 'favicon', 'arguments', 'url']);
+  if (themeFaviconPathType === 'link' && themeFaviconPathUrl) {
+    themeFaviconPath = CDAP_DIST_PATH + themeFaviconPathUrl;
```

Comment:
this doesn't seem right, asking user to add the favicon in `/cdap_dist`. Also will disappear when user upgrade.

Reply:
Seems like there's no way to allow the user to specify a file from their file system, since the route will be handled by Express... I guess we'll just have to document it